Makefile

```diff
@@ -25,9 +25,34 @@ pep8:
 	pip install -c requirements.txt pep8-naming flake8-black flake8-isort
 	flake8 src tests tests_common
 
-.PHONY: mypy
+.PHONY: mypy mypy-diff mypy-commit
+RUN_MYPY=MYPYPATH=stubs:src python -m mypy --html-report mypy -p inmanta
+
 mypy:
-	MYPYPATH=stubs:src python -m mypy --html-report mypy -p inmanta
+	$(RUN_MYPY)
+
+MYPY_TMP_FILE=.mypy-tmp
```

Comment:
this whole magic could use some docs.

Reply:
What exactly do you mean? Docs on how to use it? Some comments in the makefile explaining the magic, ...?

Reply:
I've pushed some changes regarding this comment.